jupyterhub/notebook-images/overlays/additional/elyra-notebook-imagestream.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  name: s2i-elyra-notebook
```

Comment:
Can you add similar annotations: https://github.com/opendatahub-io/odh-manifests/blob/master/jupyterhub/notebook-images/overlays/additional/minimal-notebook-imagestream.yaml#L7-L9, please?